packages/pharos/tokens/components/dropdown-multiselect.json

```diff
@@ -0,0 +1,27 @@
+{
+  "dropdown-multiselect": {
+    "color": {
+      "text": {
+        "hover": {
+          "value": "{color.hover.primary.value}"
+        },
+        "selected": {
+          "value": "{color.text.base.value}"
+        }
+      },
+      "icon": {
+        "dropdown": {
+          "value": "{color.interactive.secondary.value}"
+        }
+      }
+    },
+    "size": {
+      "height": {
+        "list": {
+          "value": "16.75rem",
+          "comment": "TODO: Update with core token value"
```

Comment:
This todo was copied from the combobox, I'm not sure if its still relevant or not